Resources/Prototypes/Recipes/Lathes/security.yml

```diff
@@ -696,3 +696,21 @@
     Plastic: 1000
     Plasma: 500
     Glass: 500
+
+- type: latheRecipe
+  id: CrimeAssist
+  result: CrimeAssistCartridge
+  completetime: 8
+  materials:
+    Glass: 600
+    Gold: 400
+    Silver: 200
```

Comment:
I am pretty sure you can use the `BaseCartridgeRecipe` here as well, even though it is defined in `cargo.yaml`.